src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  background-color: #fff;
+  transition: all 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The scale transformation on hover might cause the card to overlap with other elements. Ensure adequate spacing around the card or consider a smaller scale factor.